docs/localstack.md

```diff
@@ -16,29 +16,8 @@ Start localstack by running the command:
 $ SERVICES=dynamodb,kinesis,s3 localstack start
 ```
 
-Next we have to set up the services.
+Next we have to set up the resources:
 
-#### Create the resources table in DynamoDB:
-```shell
-$ awslocal dynamodb create-table --table-name resources \
-  --attribute-definitions AttributeName=tacoIdentifier,AttributeType=S \
-  AttributeName=externalIdentifier,AttributeType=S \
-  AttributeName=version,AttributeType=N \
-  --key-schema "AttributeName=tacoIdentifier,KeyType=HASH" \
-  --provisioned-throughput=ReadCapacityUnits=10,WriteCapacityUnits=10 \
-  --global-secondary-indexes "IndexName=ResourceByExternalIDAndVersion, \
-            KeySchema=[{AttributeName=externalIdentifier,KeyType=HASH}, \
-                       {AttributeName=version,KeyType=RANGE}], \
-            Projection={ProjectionType=ALL}, \
-            ProvisionedThroughput={ReadCapacityUnits=10,WriteCapacityUnits=10}"
-```
-
-#### Create the S3 bucket:
-```shell
-$ awslocal s3api create-bucket --bucket taco-deposited-files
 ```
-
-#### Create the Kinesis stream:
-```shell
-$ awslocal kinesis create-stream --stream-name deposit --shard-count 3
+LOCAL_ENDPOINT_HOST=localhost AWS_ACCESS_KEY_ID=999999 AWS_SECRET_ACCESS_KEY=1231 make resources
```

Comment:
One #428 gets merged this can changed to just `make resources`

Reply:
I'm going to leave this for now because I'm still seeing:
```
$ make resources
Unable to locate credentials. You can configure credentials by running "aws configure".
```

Reply:
ok - I'll look into this later